### PR TITLE
producer for testing

### PR DESCRIPTION
**Pull request by eiba**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,15 +1,28 @@
 plugins {
     id 'org.jetbrains.kotlin.jvm' version '1.3.41'
+    id 'java'
```
Comment (MartinStigen) [inline build.gradle]: remove

File: build.gradle [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
 version '1.0-SNAPSHOT'
 
 repositories {
+    maven{url "http://packages.confluent.io/maven/"}
     mavenCentral()
 }
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    compile("org.apache.kafka:kafka-clients:2.1.0")
```
Comment (MartinStigen) [inline build.gradle]: add comments/group imports

File: src/main/kotlin/Producer.kt [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+
+class Producer {
+    private val random = Random()
+    private val faker = Faker()
+
+    /**
+     * Generates and sends recordCount random records of a given topic and schema
+     */
+    fun produceData(topic: String, configFile:String,schemaFile:String,recordCount:Int,millisecondsPerRecord:Int){
```
Comment (MartinStigen) [inline src/main/kotlin/Producer.kt]: send data should be a function call and not in constructor

File: src/main/kotlin/Producer.kt [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        val props =  loadConfig(configFile)
+        val schema = Schema.Parser().parse(File(schemaFile))
+
+        KafkaProducer<Int, GenericRecord>(props).use { producer ->
```
Comment (MartinStigen) [inline src/main/kotlin/Producer.kt]: Int could be generic

File: src/main/kotlin/Util/Utils.kt [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import org.apache.kafka.clients.admin.NewTopic
+import org.apache.kafka.common.errors.TopicExistsException
+import java.io.FileInputStream
+import java.util.*
+import java.util.concurrent.ExecutionException
+
+/**
+ * Loads the config file and creates properties.
+ */
+fun loadConfig(configFile: String) = FileInputStream(configFile).use {
```
Comment (MartinStigen) [inline src/main/kotlin/Util/Utils.kt]: FileInputStream(javaClass.getResource(configFIle).file).use

File: src/main/kotlin/Producer.kt [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+                        set(field.name(),faker.bool().bool())
+                    }
+                }
+            }
+        }.build()
+    }
+}
+
+fun main(){
+    Producer().produceData("sometesttopic","./src/main/resources/conf","./src/main/resources/person.avsc",10,1000)
```
Comment (MartinStigen) [inline src/main/kotlin/Producer.kt]: change to "conf" when utils is updated

**Comment by eiba** [comment]
Thank you for your kind review! Very enlightening.

**Pull request merged**